chapter4/e2-recursion.js

```diff
@@ -0,0 +1,17 @@
+function isEven(n) {
+  if (n < 0) return isEven(-n);
```

Comment:
control structures like ifs and fors should always have curly braces. They prevent errors in the future having to do with lines ending up in/not in the conditional accidentally and make visual pattern matching easier. I recommend a bunch of if-elses for this function to make its behavior extremely explicit.